xarray/tests/test_backends_datatree.py

```diff
@@ -414,112 +446,187 @@ def test_phony_dims_warning(self, tmpdir) -> None:
                 }
 
 
-@pytest.mark.skipif(
-    have_zarr_v3, reason="datatree support for zarr 3 is not implemented yet"
-)
 @requires_zarr
+@parametrize_zarr_format
```

Comment:
nit: We can also do this, though that fixture is in `test_backends.py` : https://github.com/pydata/xarray/blob/2da7f557a75161aa193aef5314d26d61a29e90a5/xarray/tests/test_backends.py#L2281